reactDashboard/src/components/HeatMap/Heatmap.jsx

```diff
@@ -183,10 +186,11 @@ function Heatmap({
   ) {
     const counts = VALUETYPE_DEATHS === valueType ? deaths : cases;
     const total = VALUETYPE_DEATHS === valueType ? totalDeaths : totalCases;
+    const scotlandTotal = totalCount();
```

Comment:
This seems a bit clunky now that the count is part of the table body. May be cleaner to populate totalCases/totalDeaths in dataset.scotland (may already be done), and just treat it the same as the other rows

Reply:
refactored